frontend/src/components/Team/Team.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import TeamAvatar from '../../images/team.png'
+import './Team.css'
+
+const Team = () => {
+  return (
+    <section className='team'>
+        <h2>Meet our team</h2>
+        <br/>
+        <br/>
+        <div className='team__grid'>
```

Comment:
You can consider making this a reusable component to avoid code duplication.
Not sure if `<br/>` is the best tag for semantic purposes. I know there were some issues with accessibility